router/assumptions.gobra

```diff
@@ -67,6 +67,7 @@ func establishInvalidSrcAddrForTransit()
 
 ghost
 ensures noSVCBackend.ErrorMem()
+ensures noSVCBackend != nil
```

Comment:
do we really need this postcondition?

Reply:
Nice, we don't need it anymore because `packSCMP()` allows `cause == nil`.